test/VeTokenDistributor/VeTokenDistributor.spec.ts

```diff
@@ -0,0 +1,526 @@
+import chai, { expect } from "chai"
+import { solidity } from "ethereum-waffle"
+import { parseEther, parseUnits } from "ethers/lib/utils"
+import { ethers, waffle } from "hardhat"
+import { TestERC20, VePERP, VeTokenDistributor, VeTokenDistributor__factory } from "../../typechain"
+import { getLatestTimestamp, getWeekTimestamp } from "../shared/utilities"
+
+chai.use(solidity)
+
+describe("VeTokenDistributor", () => {
+    const [admin, alice, bob, carol] = waffle.provider.getWallets()
+    let vePERP: VePERP
+    let veTokenDistributor: VeTokenDistributor
+    let testPERP: TestERC20
+    let testUSDC: TestERC20
+    const DAY = 86400
+    const WEEK = DAY * 7
+    const MONTH = DAY * 30
+    const YEAR = DAY * 365
+    const MIN_LOCK_DURATION = 4 * WEEK
+
+    beforeEach(async () => {
+        const testERC20Factory = await ethers.getContractFactory("TestERC20")
+        testPERP = await testERC20Factory.deploy()
+        await testPERP.__TestERC20_init("PERP", "PERP", 18)
+
+        testUSDC = await testERC20Factory.deploy()
+        await testUSDC.__TestERC20_init("USDC", "USDC", 18)
```

Comment:
Suggest changing the decimals back to 6 since it is still called test"USDC", or changing the tokens name to avoid confusion

Reply:
thanks ~ didn't notice this

Reply:
fixed